src/configLoaders/DefaultConfigLoader.ts

```diff
@@ -0,0 +1,20 @@
+import ConfigLoader from './ConfigLoader';
+import * as path from 'path';
+
+export default class DefaultConfigEditor implements ConfigLoader {
+  private fs: any;
+  private projectRoot: string;
+
+  constructor(projectRoot: string, fs: any) {
+    this.projectRoot = projectRoot;
+    this.fs = fs;
+  }
+
+  public loadConfig(): string {
+    return JSON.stringify(this.getJestConfigFromPackageJson(this.projectRoot));
+  }
+
+  private getJestConfigFromPackageJson(projectRoot: string) {
```

Comment:
The fact that you need two methods to do this is an indicator that it's too complex.
This line for example: `JSON.parse(this.fs.readFileSync(path.join(projectRoot, 'package.json'), 'utf8')).jest;` I have no clue what the object is that we're calling `jest` on